nexus/tests/unit/test_qmcpack_input.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python
+def test_generate_kspace_jastrow():
```

Comment:
Similarly repackage into nxs-test with the contents of "main" and all other local functions going into a new function called "qmcpack_input".